### PR TITLE
(RE-4155) Create directorires with 775 perms

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

**Comment by haus** [comment]
@melissa better. as long as we're going down this road, let's make it a variable and use it to define artifact_dir earlier in the task.

**Comment by melissa** [comment]
@haus better?

File: tasks/ship.rake [hunk 1]
```diff
@@ -154,7 +154,8 @@ namespace :pl do
       Pkg::Util::RakeUtils.invoke_task("pl:fetch")
       target = args.target || "artifacts"
       local_dir = args.local_dir || "pkg"
-      artifact_dir = "#{Pkg::Config.jenkins_repo_path}/#{Pkg::Config.project}/#{Pkg::Config.ref}/#{target}"
+      parent_dir = "#{Pkg::Config.jenkins_repo_path}/#{Pkg::Config.project}/#{Pkg::Config.ref}"
```
Comment (haus) [inline tasks/ship.rake]: how about project_basedir instead of parent_dir?

**Comment by haus** [comment]
:+1:

**Comment by shrug** [comment]
:+1:  I'm happy if travis is happy

**Comment by melissa** [comment]
@shrug Travis is happy

**Pull request merged**